src/components/tile/__tests__/tile-test.js

```diff
@@ -0,0 +1,19 @@
+/** @jsx React.DOM */
+
+jest.dontMock('../code/views/tileView.jsx');
+
+var React = require('react/addons');
+var TileComponent = require('../code/views/tileView.jsx');
+var TestUtils = React.addons.TestUtils;
+
+describe('TileComponent', function() {
+
+  it('is an element', function() {
+    var img = {
+      src: 'foo',
+      alt: 'bar'
+    };
+    expect(TestUtils.isElement(<TileComponent image={img} title='title' />)).toBeTruthy();
```

Comment:
We need to refactor this. Let's chat about `this.children` in the next standup. You don't pass components (generally) in as properties. Instead, you'll nest them:

```jsx
<TileComponent title="title">
    <ImageComponent src="blah.jpg" alt="blah"/>
</TileComponent>
```